src/stp/plays/referee_specific/FreeKickThem.cpp

```diff
@@ -2,11 +2,11 @@
 // Created by jordi on 07-05-20.
 //
 
-#include "stp/plays/FreeKickThem.h"
+#include "include/roboteam_ai/stp/plays/referee_specific/FreeKickThem.h"
 
 #include "stp/invariants/game_states/FreeKickThemGameStateInvariant.h"
-#include "stp/roles/Defender.h"
-#include "stp/roles/Formation.h"
+#include "include/roboteam_ai/stp/roles/passive/Defender.h"
+#include "include/roboteam_ai/stp/roles/passive/Formation.h"
```

Comment:
I don't like these includes. Why does it add `include/roboteam_ai`, it works the line above it.